frontend/src/containers/Review/ReviewControl/ReviewControl.js

```diff
@@ -1,35 +1,93 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
 import { Form, Button } from "react-bootstrap";
-import {
-    PaperSpec,
-} from "../../../components";
+import { PaperSpec } from "../../../components";
+import { reviewActions, paperActions } from "../../../store/actions";
 import "./ReviewControl.css";
 
+
 class ReviewControl extends Component {
     constructor(props) {
         super(props);
         this.state = {
             title: "",
-            id: 5,
             content: "",
-            paper: {
-                id: 1,
-                title: "paper_title",
-                abstract: "abstract abstract abstract abstract abstract abstract abstract abstract ",
-                date: "2019.10.30.",
-                authors: "paper_authors",
-                keywords: [{ name: "paper_keywords", type: "author" }],
-                likeCount: 101,
-                reviewCount: 3,
-            },
+            paper: { },
+            likeCount: 0,
+            authorNames: [],
+            // keywords: [],
+            date: "",
+            thisReview: { id: 0 },
+            paperId: 0,
         };
         this.handleChange = this.handleChange.bind(this);
         this.clickButtonHandler = this.clickButtonHandler.bind(this);
     }
 
+    componentDidMount() {
+        if (this.props.mode === 0) {
+            this.props.onGetPaper({ id: this.props.match.params.paper_id })
+                .then(() => {
+                    this.setState({
+                        paper: this.props.selectedPaper,
+                        likeCount: this.props.selectedPaper.count.likes,
+                        authorNames: this.props.selectedPaper.authors.map((x) => `${x.first_name} ${x.last_name}`),
```

Comment:
This part is wrong, because PaperSpec get an array of author object.

Reply:
Oh, I think PaperSpec would get just author names. 
<h3 id="authors">{this.props.authors}</h3>
Related things are covered like this in the current PaperSpec.
Would it be modified?

Reply:
@kosumin Well, since my last PR is merged, it changed. Please merge the latest master version.

Reply:
Do you mean not the array of author names(string), but the array of author objects(object), right?

Reply:
@kosumin Yes.

Reply:
I'm sorry to keep asking. But I could not understand why there should be an array of objects, instead of an array of strings, since there's no way to handle author objects in the current PaperSpec, but there is only a function to manage keywords. And even when I changed authorNames to an array of author objects, error occurred.

Reply:
Anyway, we need author objects for Paperspec, right? Then I would add function for handle author objects!

Reply:
@kosumin I finally knew why this situation happened! Didn’t we talk about this in S-lab??? Because of it, maybe I misunderstood, sorry. I thought you mentioned you would fix about authors in PaperSpec, and about keywords, I thought you told me that I should fix it. That’s why I expected you reflect it. Plus, I thought handling authors in PaperSpec is already merged, sorry.

Then, about what did you mentioned in S-lab? Anyway, it would be good if you fix it. I just let it be because I thought you told you would do that.